src/internal/faucet.ts

```diff
@@ -11,28 +11,28 @@
 import { AptosConfig } from "../api/aptos_config";
 import { postAptosFaucet } from "../client";
 import { AccountAddress } from "../core";
-import { HexInput, TransactionResponse } from "../types";
+import { TransactionResponse } from "../types";
 import { DEFAULT_TXN_TIMEOUT_SEC } from "../utils/const";
 import { waitForTransaction } from "./transaction";
 
 export async function fundAccount(args: {
   aptosConfig: AptosConfig;
-  accountAddress: HexInput;
+  accountAddress: AccountAddress;
   amount: number;
   timeoutSecs?: number;
 }): Promise<Array<string>> {
   const { aptosConfig, accountAddress, amount } = args;
   const timeoutSecs = args.timeoutSecs ?? DEFAULT_TXN_TIMEOUT_SEC;
   const { data } = await postAptosFaucet<any, Array<string>>({
     aptosConfig,
+    body: null,
     path: "mint",
```

Comment:
Let's land Greg's stuff first? It switches us to `fund`.

Reply:
Updated to use fund